features/docs/gherkin/empty_scenario.feature

```diff
@@ -0,0 +1,73 @@
+Feature: Empty scenario
+
+  A scenario can be empty.
+  Background and hooks are not executed.
```

Comment:
Is this desirable / agreed upon across all flavours. Feels like something semi big to discuss.

P.S. I'm 100% for this. Just wanted to bring it up.

Reply:
This has already been discussed a while ago here: https://github.com/cucumber/cucumber/issues/249
In order to properly close the issue, we have agree to open a PR to add an empty scenario in the compatibility kit: https://github.com/cucumber/cucumber/pull/1498

Actually it looks like at least cucumber-ruby and cucumber-js are really close to the described behavior. It is just a matter of standardization and fixes like the actual PR